<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_popup_Purchase_Inquiry</class>
 <widget class="QMainWindow" name="frm_popup_Purchase_Inquiry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>询价</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbtn_Cancel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_Save">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_num">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数量</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_price">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>总价</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cycle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>供货周期</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_num">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_price">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cycle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_Comments">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Comments">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>备注</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
